tests/no_parallel.rs

```diff
@@ -0,0 +1,14 @@
+#![cfg(not(feature = "parallel"))]
```

Comment:
@azriel91 addresses https://github.com/amethyst/specs/pull/674#pullrequestreview-352585509 - thanks for the feedback!